fw/http_sched_ratio.c

```diff
@@ -1047,6 +1048,8 @@ tfw_sched_ratio_srvdesc_setup_srv(TfwServer *srv, TfwRatioSrvDesc *srvdesc)
 
 	conn = srvdesc->conn;
 	list_for_each_entry(srv_conn, &srv->conn_list, list) {
+		if (tfw_srv_conn_unscheduled(srv_conn))
+			continue;
```

Comment:
It seems we don't need `tfw_srv_conn_unscheduled()` calls in the schedulers and can just totally revert all scheduler changes.

Reply:
See https://github.com/tempesta-tech/tempesta/pull/1595/files#r857456928